app/components/icons/library.tsx

```diff
@@ -1051,3 +1051,29 @@ export const HelpIcon = (props: SVGProps<SVGSVGElement>) => (
     </defs>
   </svg>
 );
+
+export const DuplicateIcon = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    width="20"
+    height="20"
+    viewBox="0 0 20 20"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M1.66669 4.33329C1.66669 3.39987 1.66669 2.93316 1.84834 2.57664C2.00813 2.26304 2.2631 2.00807 2.5767 1.84828C2.93322 1.66663 3.39993 1.66663 4.33335 1.66663H10.6667C11.6001 1.66663 12.0668 1.66663 12.4233 1.84828C12.7369 2.00807 12.9919 2.26304 13.1517 2.57664C13.3334 2.93316 13.3334 3.39987 13.3334 4.33329V10.6666C13.3334 11.6 13.3334 12.0668 13.1517 12.4233C12.9919 12.7369 12.7369 12.9918 12.4233 13.1516C12.0668 13.3333 11.6001 13.3333 10.6667 13.3333H4.33335C3.39993 13.3333 2.93322 13.3333 2.5767 13.1516C2.2631 12.9918 2.00813 12.7369 1.84834 12.4233C1.66669 12.0668 1.66669 11.6 1.66669 10.6666V4.33329Z"
+      stroke="#667085"
```

Comment:
How we do the icons is that all the colors get replaced by `currentColor`, this way the svg inherits the color of the parent and it makes it very easy to use 1 svg with different colors without having to pass special props and so on.